src/router/index.ts

```diff
@@ -45,6 +48,11 @@ const routes: Array<RouteRecordRaw> = [
             name: "settings",
             component: Settings,
           },
+          {
+            path: "my-feed",
```

Comment:
lets have the path as just `feed`

Reply:
This would be for user feed and then we might have another route for other users feed,

Reply:
okay deferring to Leif's guidance here